cdap-cli/src/main/java/co/cask/cdap/cli/command/metadata/AddMetadataPropertiesCommand.java

```diff
@@ -34,6 +34,10 @@
 
   private final MetadataClient client;
 
+  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' is of the form " +
+    "'<entity-type>:<namespace-id>.<entity-name>', where '<entity-type>' is one of " +
```

Comment:
What about this (as your example had `entity-id` twice, once as the resolved `ENTITY` and once elsewhere): use `<entity>` instead, and something such as (with some line breaks added here for readability, though they would not appear in the final:
```
public static final String ENTITY_DESCRIPTION_STRING = String.format("

'<entity>' is of the form '<entity-type>:<entity-id>', where '<entity-type>' is one of
'artifact', 'app', 'program', 'dataset', 'stream', or 'view'.

For artifacts and apps, '<entity-id>' is composed of the namespace, entity name, and
version, such as '<namespace-name>.<artifact-name>.<artifact-version>' or
'<namespace-name>.<app-name>.<app-version>'.

For programs, '<entity-id>' includes the application name and the program type:
'<namespace-name>.<app-name>.<program-type>.<program-name>'.

For datasets and streams, '<entity-id>' is the namespace and entity names, such as
'<namespace-name>.<dataset-name>' or '<namespace-name>.<stream-name>'.

For (stream) views, '<entity-id>' includes the stream that they were created from:
'<namespace-name>.<stream-name>.<view-name>'.

'<program-type>' is one of flow, mapreduce, service, spark, worker, or workflow.");
```